doors_dashboards/components/constant.py

```diff
@@ -7,4 +7,7 @@
 HEADER_BGCOLOR = '#3B4758'
 CONTAINER_BGCOLOR = '#161B21'
 FONT_COLOR = "#FFFFFF"
-FONT_FAMILY = "Roboto, Helvetica, Arial, sans-serif"
\ No newline at end of file
+FONT_FAMILY = "Roboto, Helvetica, Arial, sans-serif"
+FONT_SIZE_NUMBER = '25px'
+FONT_SIZE = 'x-large'
+SCATTER_FONT_SIZE = '18'
```

Comment:
There is just one module where SCATTER_FONT_SIZE is needed, and there it is not even used. We should get rid of it.

Reply:
I made this its not being used in this because I was suppose to use it later on. which I have now used it. Those changes are not committed in this pull request where I am using this.

Reply:
Okay. Still, I think that this sort of constant that is not shared across modules should be in the scatterplot module

Reply:
Ok, agreed!!